hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -165,6 +165,12 @@
           + "used for pruning files during the index lookups. Only applies if "
           + ENABLE_METADATA_INDEX_COLUMN_STATS.key() + " is enabled.A");
 
+  public static final ConfigProperty<Integer> COLUMN_STATS_INDEX_PARALLELISM = ConfigProperty
+          .key(METADATA_PREFIX + ".index.column.stats.parallelism")
+          .defaultValue(1)
```

Comment:
why 1. can we make this 10 may be.

Reply:
done 10 for now.. but i think once we run perf tests we'll be able to come up with better default value.

Reply:
sounds good!